### PR TITLE
app-misc/hyfetch: enable py3.13, remove patch

### PR DESCRIPTION
**Pull request by BKasin**
Upstream has implemented what the patch accomplished, so remove.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-misc/hyfetch/hyfetch-9999.ebuild [hunk 1]
```diff
@@ -40,6 +35,8 @@ python_install() {
 	cp neofetch "${D}/usr/bin/neowofetch" || die
 
 	rm -r "${D}/usr/lib/${EPYTHON}/site-packages/hyfetch/scripts" || die
```
Comment (thesamesam) [inline app-misc/hyfetch/hyfetch-9999.ebuild:37]: Can you fix this to use the functions from https://devmanual.gentoo.org/eclass-reference/python-utils-r1.eclass/index.html?

Comment (BKasin) [reply app-misc/hyfetch/hyfetch-9999.ebuild:37]: I'll do that today. Apologies, had to be away from computer last week.

Comment (thesamesam) [reply app-misc/hyfetch/hyfetch-9999.ebuild:37]: No worries, thanks!

**Review by juippis — COMMENTED** [review]
```
 * 
 * QA Notice: The following unexpected files/directories were found
 * top-level in the site-packages directory:
 * 
 *   /usr/lib/python3.10/site-packages/docs
 *   /usr/lib/python3.11/site-packages/docs
 *   /usr/lib/python3.12/site-packages/docs
 * 
 * This is most likely a bug in the build system.  More information
 * can be found in the Python Guide:
 * https://projects.gentoo.org/python/guide/qawarn.html#stray-top-level-files-in-site-packages
```

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**